polynote-frontend/polynote/ui/component/plot_editor.ts

```diff
@@ -1,5 +1,6 @@
 'use strict';
 
+import * as _ from "lodash";
```

Comment:
Is this new dependency necessary?

Reply:
Maybe not, this is only used in two places:

* L352: `!_.find(this.yMeasures, measureConfig)`
* L449: `_.map(batch, (obj: object) => _.mapValues(obj, normalizeValues))`

Those can be probably be rewritten in vanilla JS, but I'm no JS expert and am under the impression that comparing objects (and some object-related operations) are non-trivial to write correctly in JS.

I 'll try to implement those methods by hand.

Reply:
Dependency removed in a004e8f